test/test-balanced-builder.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-env mocha */
+'use strict'
+
+const expect = require('chai').expect
+const pull = require('pull-stream')
+
+const builder = require('../src/builder/balanced')
+
+function reduce (leaves, callback) {
+  if (leaves.length > 1) {
+    callback(null, { children: leaves })
+  } else {
+    callback(null, leaves[0])
+  }
+}
+
+const options = {
+  maxChildrenPerNode: 3
+}
+
+describe('balanced builder', () => {
+  it('reduces one value into itself', callback => {
+    pull(
+      pull.values([1]),
+      builder(reduce, options),
+      pull.collect((err, result) => {
+        expect(err).to.not.exist
+        expect(result).to.be.eql([1])
+        callback()
+      })
+    )
+  })
+
+  it('reduces 3 values into parent', callback => {
```

Comment:
On the other hand, given this https://github.com/feross/eslint-config-standard/pull/54#issuecomment-262828198 gigantic history I don't feel like touching it tbh